src/triggers/missing.ts

```diff
@@ -42,6 +43,21 @@ export const missingMetadataIPFSHash: Trigger<undefined> = async (clients) => {
   return nfts;
 };
 
+export const missingMimeType: (source: SourceIPFS) => Trigger<undefined> = (source) => {
```

Comment:
lets put this in a folder with the processor. actually in general i think we should be keeping triggers together with their processors in the same file or folder, and refactor towards that direction since there's very little sharing of triggers across processors.